core/src/test/java/fr/sncf/osrd/envelope_sim_infra/MRSPTest.java

```diff
@@ -125,14 +127,82 @@ public void mrspFromInfraPath() {
                 new TrackRangeView(15, 80, infra.getEdge("track", Direction.FORWARD))
         );
         var testRollingStock = TestTrains.VERY_SHORT_FAST_TRAIN;
-
-        var mrsp = MRSP.from(path, testRollingStock, true, null);
+        var driverBehaviour = new DriverBehaviour(0, 0);
+        var mrsp = MRSP.from(path, testRollingStock, true, null, driverBehaviour);
```

Comment:
```suggestion
        var mrsp = MRSP.from(path, testRollingStock, true, null);
```
There's no need to call the from() with a driver behavior as we implemented a specific method that does it